tests/test_rtdc_feat_anc_plugin.py

```diff
@@ -905,6 +958,160 @@ def test_pf_wrong_length_2():
+        image_gauss_filter = gaussian_filter(ds["image"], sigma=(0, 1, 1))
+    with RTDCWriter(h5path) as hw:
+        hw.store_feature("image_gauss_filter", image_gauss_filter)
+    with h5py.File(h5path, mode="r") as h5:
+        events = h5["events"]
+        assert "image_gauss_filter" in events.keys()
+        feat = h5["events"]['image_gauss_filter']
+        fshape = feat.shape
+        assert feat.ndim == 3
+        assert fshape[1] > 3 and fshape[2] > 3
+        assert b'CLASS' in feat.attrs.keys()
+        assert b'IMAGE_SUBCLASS' in feat.attrs.keys()
+        assert b'IMAGE' in feat.attrs.get('CLASS')
+
+
+@pytest.mark.filterwarnings(
+    "ignore::dclab.rtdc_dataset.config.WrongConfigurationTypeWarning")
+def test_pf_store_image_plugin_data_without_shape_issue_171():
+    """Test storage of image plugin feature attributes without shape info"""
+    h5path = retrieve_data("fmt-hdf5_fl_2018.zip")
+    info = example_plugin_info_non_scalar_feature()
+    PlugInFeature("image_gauss_filter", info)
+    with dclab.new_dataset(h5path) as ds:
+        image_gauss_filter = gaussian_filter(ds["image"], sigma=(0, 1, 1))
+    with RTDCWriter(h5path) as hw:
+        hw.store_feature("image_gauss_filter", image_gauss_filter)
+    with h5py.File(h5path, mode="r") as h5:
+        events = h5["events"]
+        assert "image_gauss_filter" in events.keys()
+        feat = h5["events"]['image_gauss_filter']
+        fshape = feat.shape
+        assert feat.ndim == 3
+        assert fshape[1] > 3 and fshape[2] > 3
+        assert b'CLASS' in feat.attrs.keys()
+        assert b'IMAGE_SUBCLASS' in feat.attrs.keys()
+        assert b'IMAGE' in feat.attrs.get('CLASS')
+
+
+@pytest.mark.filterwarnings(
+    "ignore::dclab.rtdc_dataset.config.WrongConfigurationTypeWarning")
+def test_pf_store_mask_plugin_data_with_shape_issue_171():
+    """Test storage of mask plugin feature attributes with shape info"""
+    h5path = retrieve_data("fmt-hdf5_fl_2018.zip")
+    info = example_plugin_info_mask_feature()
+    # Mention feature shape in the info
+    info["feature shapes"] = [(80, 250)]
+    PlugInFeature("mask_feat", info)
+    with dclab.new_dataset(h5path) as ds:
+        mask_feat = np.logical_not(ds["mask"])
+    with RTDCWriter(h5path) as hw:
+        hw.store_feature("mask_feat", mask_feat)
+    with h5py.File(h5path, mode="r") as h5:
+        events = h5["events"]
+        assert "mask_feat" in events.keys()
+        feat = h5["events"]['mask_feat']
+        fshape = feat.shape
+        assert feat.ndim == 3
+        assert fshape[1] > 3 and fshape[2] > 3
+        assert b'CLASS' in feat.attrs.keys()
+        assert b'IMAGE_SUBCLASS' in feat.attrs.keys()
+        assert b'IMAGE' in feat.attrs.get('CLASS')
+
+
+@pytest.mark.filterwarnings(
+    "ignore::dclab.rtdc_dataset.config.WrongConfigurationTypeWarning")
+def test_pf_store_mask_plugin_data_without_shape_issue_171():
+    """Test storage of mask plugin feature attributes without shape info"""
```

Comment:
without shape info? You are setting it in line 1044

Reply:
I dont get your point here. I am just passing an argument `shape` of the feature in `store_feature()`.

Reply:
In [line 1044](https://github.com/RaghavaAlajangi/dclab/blob/master/tests/test_rtdc_feat_anc_plugin.py#L1044) you are passing shape info, so the description is not right imo

Reply:
Ha..I get it. Thanks

Reply:
I changed the function name to address this (added `shape_in_info` at the end of function)

Reply:
I changed the function name accordingly (added `shape_in_info()`). Is it clear enough?